axelor-base/src/main/java/com/axelor/apps/base/service/administration/SequenceService.java

```diff
@@ -244,7 +245,7 @@ protected String computeNextSeq(
           StringUtils.leftPad(
               sequenceVersion.getNextNum().toString(), sequence.getPadding(), PADDING_STRING);
     } else {
-      sequenceValue = findNextLetterSequence(sequenceVersion);
+      sequenceValue = findNextLetterSequence(sequenceVersion, sequence.getPadding());
```

Comment:
Actually I think we should use `StringUtils.leftPad`as it's done above for numbers. Could you check if it's doable?